reference/audit_logs.rst

```diff
@@ -0,0 +1,241 @@
+- **user-tag**:         The user making the request.
+- **model**:            The model a request is acting on, can be empty for controller level requests.
+- **is-response**:      Indicates whether this log is for a request or response message.
+- **params**:           Populated during requests with any request parameters
+- **errors**:           Populated during responses with any response errors.
+
+It's important to note that JIMM logs requests and responses separately for audit logs and understanding 
+how to associate a request with a response is very helpful. This can be done using the `conversation-id` and `message-id` fields.
+When a client establishes a connection with JIMM and begins making requests, a unique `conversation-id` is generated for 
+the lifetime of that websocket connection and each request/response pair will have the same `message-id`, which itself will
+increment whenever a new request is made. Then observing the `is-response` (boolean) and `errors` fields, one can ascertain whether 
+a call was successful.
+
+An example of a request and response audit log are below.
+The first is a request for a model status and the second is the response.
+Some interesting things to note:
+
+- The message-id is 2 because the first message (not shown) was a login request.
+- The response does not include the response payload (i.e. the application status), this information is not logged.
+- The request includes a "patterns" parameter that is set to null, this is specific to the `status` call.
+- The facade information is only included on the request.
+
+.. code:: yaml
+
+    - time: 2023-09-06T10:21:16.72Z
+    conversation-id: "b501bba5508367e5"
+    message-id: 2
+    facade-name: Client
+    facade-method: FullStatus
+    facade-version: 6
+    object-id: ""
+    user-tag: user-alice@external
+    model: controller-1/test-model
+    is-response: false
+    params:
+        patterns: null 
+
+    - time: 2023-09-06T10:21:16.797Z
+    conversation-id: "b501bba5508367e5"
+    message-id: 2
+    facade-name: ""
+    facade-method: ""
+    facade-version: 0
+    object-id: ""
+    user-tag: user-alicen@external
+    model: controller-1/test-model
+    is-response: true
+    params: {}
+    errors:
+        results:
+        - error:
+            code: ""
+            message: ""
+
+Pagination
+~~~~~~~~~~
+
+``--offset``
+    offset the set of returned audit events
+``--limit``
+    limit the maximum number of returned audit events
+
+Audit logs are returned in a paginated set, by default, of size 50. 
+The size of each page can be increased to a maximum of 1000 using the
+``--limit`` flag.
+
+Paging through the result set is also possible with the ``--offset`` flag.
+
+Get second page::
+
+    jimmctl audit-events --offset 50
+
+Change the page size to 100 and get the third page::
+
+    jimmctl audit-events --offset <(page_number-1)*100> --limit 100
+
+Time filters
+~~~~~~~~~~~~
+
+``--after``
```

Comment:
Again let's explain here's available filters, and then have the pagination, time, method, model and user filters nested inside

Reply:
Not sure I captured the intent of this one correctly. Let me know.